src/sqlfluff/core/default_config.cfg

```diff
@@ -6,10 +6,13 @@ templater = jinja
 rules = None
 exclude_rules = None
 recurse = 0
+# Below controls SQLFluff output, see max_line_length for SQL output
 output_line_length = 80
+# Number of passes to run before admitting defeat
 runaway_limit = 10
```

Comment:
Added comments for config options that are not obvious

Reply:
might be worth doing a small comment above each of the options here for completeness so users know what the values can be set to without asking us?

e.g. `verbose is an integer indicating the level of log output.`
`rules is a comma separated list of rules to include in linting/fixing. If set to None the all rules are included.`
etc.

Given that we show this file in the docs I think a lot of people use it as a tldr reference for what options are available.

Reply:
Added some more descriptions there in https://github.com/sqlfluff/sqlfluff/pull/2347/commits/85542ad13e7984e0be2be0deb07dd1269a8791b8

Reply:
Looks great thanks :)